runtime/doc/lsp.txt

```diff
@@ -228,6 +229,27 @@ To see the capabilities for a given server, try this in a LSP-enabled buffer: >v
 
     :lua =vim.lsp.get_clients()[1].server_capabilities
 
+================================================================================
+ROOT MARKERS                                            *lsp-root-markers*
+
+In `vim.lsp.Config.root_markers` it's possibile to specify marker's
+priority. The default behaviour performs a folder-up first search, but it can
+be configured to look for multple markers in each directory before going up.
+
+Example (default, folder-up first):
+
+    root_markers = { "stylua.toml", ".git" }
+
+Looks for `stylua.toml` in every folder going up, if it isn't found it does
+the same for `.git`.
+
+Example (mixed priorities):
+
+    root_markers = { { "stylua.toml", ".luarc.json" }, { ".git" } }
```

Comment:
```suggestion
    root_markers = { { 'stylua.toml', '.luarc.json' }, '.git' }
```

This show that each element can be a `string[]` or a `string`.

Reply:
Should I commit a fixup?

Reply:
Can do, but will need to squash before we merge.